### PR TITLE
Skip functional tests until https://github.com/aspnet/AspNetCore/issues/8439 is resolved

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Review by analogrelay — APPROVED** [review]

File: src/MusicStore/test/MusicStore.E2ETests/MusicStore.E2ETests.csproj [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <NoWarn>$(NoWarn);NU1605</NoWarn>
     <!-- https://github.com/aspnet/AspNetCore/issues/6549 -->
     <BuildHelixPayload>false</BuildHelixPayload>
+    <IsTestProject>false</IsTestProject>
```
Comment (anurse) [inline src/MusicStore/test/MusicStore.E2ETests/MusicStore.E2ETests.csproj]: Put a comment in?

Comment (pranavkm) [reply src/MusicStore/test/MusicStore.E2ETests/MusicStore.E2ETests.csproj]: I have the follow ready - https://github.com/aspnet/AspNetCore/pull/8482. Hopefully this doesn't stay skipped very long

Comment (natemcmaster) [reply src/MusicStore/test/MusicStore.E2ETests/MusicStore.E2ETests.csproj]: For future reference, comments would have helped. It's May 31 and this test is still disabled, and it took me some digging to find the reason.

Comment (pranavkm) [reply src/MusicStore/test/MusicStore.E2ETests/MusicStore.E2ETests.csproj]: Oops

**Pull request merged**